cogs/discord_datamining.py

```diff
@@ -74,18 +74,30 @@ async def commit_poster_loop(self, embed_title:str, api_url:str, repo_url:str):
                 data = await r.json()
             comment_text.extend([(i['commit_id'], re.sub(r"^## (.+)", r"**\1**", i['body'])) for i in data])
         self.logger.info(f"Logging info of {len(comment_text)} comments")
-
-        # Format it all into an embed
+        #get the description
+        pog =  '\n'.join([f"{i['commit']['message']} - [Link]({repo_url}commit/{i['sha']})" for i in new_commits])
```

Comment:
maybe make a more descriptive variable name

Reply:
why dont you do it
all my variables in c# are just called bob

Reply:
Yes but i’d like to keep these variable names somewhat descriptive, “pog” doesn’t give me anything.

Reply:
I think i made them more descriptive.